### ISSUE TITLE
Getting an issue while integrating my project with btcpay 

### ISSUE THREAD

**Issue by Saurabh130** [btcpayserver/btcpay-python#9]
```
raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='mybtcpay.local', port=443): Max retries exceeded with url: /tokens (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645)'),)) 
```

I am getting this error followed by other traceback error during custom integration while I run this -
```
client = BTCPayClient.create_client(host='https://btcpay.example.com', code=<pairing-code>)
```

**Comment by wiz** [comment]
This is because the *.local is a self-signed SSL certificate. You need to manually trust it or disable verification in your app.

**Comment by NicolasDorier** [comment]
As I said, don't use HTTPS, use HTTP instead.

**Comment by JeffVandrewJr** [comment]
Closing this issue since it doesn't relate to the Python library specifically. If you need additional help on this issue, chat.btcpayserver.org may be a better venue.

**Comment by NicolasDorier** [comment]
@JeffVandrewJr I think he had another issue by trying with HTTP

**Comment by Saurabh130** [comment]
@JeffVandrewJr while using http its giving client error : bad request

**Comment by NicolasDorier** [comment]
Probably related to this part: `NOTE THAT PAIRING CODES WORK ONLY ONCE`

**Comment by NicolasDorier** [comment]
@JeffVandrewJr I think it would be cool to have Legacy API Key as well. It is easier than a keypair for noobs.

**Comment by Saurabh130** [comment]
Should I delete the store and recreate it?

**Comment by NicolasDorier** [comment]
you should get a new pairing code
